packages/paste-website/src/pages/utilities/aspect-ratio/index.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Aspect Ratio - Utilities
+package: '@twilio-paste/aspect-ratio'
+description: The Aspect Ratio utility is used to embed images, video, media, components, or other HTML elements. It prevents skewing, cropping, etc.
+---
+
+import {graphql} from 'gatsby';
+import Img from 'gatsby-image';
+import {Anchor} from '@twilio-paste/anchor';
+import {Absolute} from '@twilio-paste/absolute';
+import {AspectRatio} from '@twilio-paste/aspect-ratio';
+import {Box} from '@twilio-paste/box';
+import Changelog from '@twilio-paste/aspect-ratio/CHANGELOG.md';
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {SidebarCategoryRoutes} from '../../../constants';
+
+export const pageQuery = graphql`
+  {
+    allPasteUtility(filter: {name: {eq: "@twilio-paste/aspect-ratio"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    file(sourceInstanceName: {eq: "assets"}, relativePath: {eq: "images/anchor-dont-1@4x.png"}) {
+      childImageSharp {
+        fluid(maxWidth: 500) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Aspect Ratio"
+  categoryRoute={SidebarCategoryRoutes.UTILITIES}
+  githubUrl="https://github.com/twilio-labs/paste/tree/master/packages/paste-core/utilities/aspect-ratio"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/utilities-aspect-ratio--default"
+  data={props.data.allPasteUtility.edges}
+/>
+
+***
+
+<content>
+
+## Guidelines
+
+### About Aspect Ratio
+
+The Aspect Ratio component dynamically modifies the height of an element based on its
+current width to match a height-to-width ratio. For example, a 4:3 ratio would make
```

Comment:
"width-to-height" instead of "height-to-width"? to match the ratio we use

Reply:
👍